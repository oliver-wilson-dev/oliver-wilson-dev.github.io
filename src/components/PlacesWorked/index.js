/* eslint-disable max-len */
import React from 'react';
import styles from './index.css';
import AsosLogo from './asos-logo.svg';
import ItvLogo from './itv-logo.svg';
import Section from '../Section';

const PlacesWorked = () => (
  <section className={styles.sectionWrapper}>
    <Section>
      <h3 className={styles.statement}>
        Proudly contributing towards projects in the fashion and television industries for over three years.
      </h3>
      <div className={styles.companyLogoWrapper}>
        <AsosLogo className={styles.companyLogo} />
        <ItvLogo className={styles.itvLogo} />
      </div>
    </Section>
  </section>
);

export default PlacesWorked;
