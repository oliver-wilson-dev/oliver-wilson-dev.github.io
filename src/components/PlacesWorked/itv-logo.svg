<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1"
	 id="svg2" x="0px" y="0px"
	 viewBox="0 0 447 223.7" style="enable-background:new 0 0 447 223.7;">
<path id="path3018" class="st0" d="M0,29.4C0,13.2,11.4,0,29.8,0
	c19.4,0,29.7,13.9,29.7,30.1c0,17.9-13.1,28.8-29.6,28.8C12,58.9,0,46.7,0,29.4z"/>
<path id="path3020" class="st0" d="M0.2,83.4
	c0-2.7,1.5-4.7,4.6-4.7H55c2.7,0,4.4,2.1,4.4,4.7v77c0,13.1,7.5,28.4,30.6,28.4c22.5,0,34.1-14,34.1-42.5v-6l23.9,45.4l3.1,15.4
	c0,0-23.4,22.6-65.8,22.6c-21.1,0-85-3.5-85-73.4V83.4H0.2z"/>
<path class="st1" d="M305.5,138.6l-33.7-16.8c0,0,6.4,10.5,6.4,23.8c0,26.1-20.9,42.9-50.6,42.9c-32.5,0-45.1-19.9-45.1-45.9v-10.8
	l0,0c0-0.3,0-0.5,0-0.8v-18.5c0,0,8.2,3.1,16.2,3.1h30.6c3,0,4.1-2,4.1-4.6V83.7c0-3.1-1.3-5-4.6-5h-46.2V33
	c0-7.6-2.9-17.8-28.9-17.8c-23.9,0-29.6,7.7-29.6,16.5V142c0,38.5,26.9,59.1,26.9,59.1s22.5,22.4,71.8,22.4c63.2,0,85.6-44,85.6-44
	L305.5,138.6z"/>
<path class="st2" d="M443.3,85l-25.6-7.9c-4.1-1.1-6.9-0.2-7.9,3.3c0,0-15.1,51.4-38.8,94.5c-37.3-68.6-51.9-90-51.9-90l0,0
	c-3.6-6-8.7-9-14.5-9c-8.7,0-19.4,4.3-22.4,6.2c-13.8,8.6-17.8,14.3-17.9,22.1c0,5.3,1.8,8,4.2,12.5l39.8,62.8l19.2,29
	c6.1,9.1,12.2,15,29.6,15c21.6,0,28.2-8.1,34.6-17.6c0,0,33.3-51,54.6-112.9C448,87.4,446.4,86.1,443.3,85z"/>
</svg>
