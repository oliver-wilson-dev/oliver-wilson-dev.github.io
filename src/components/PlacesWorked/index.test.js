import React from 'react';
import { shallow } from 'enzyme';
import PlacesWorked from '.';

jest.mock('../Section', () => {
  const Section = () => null;

  return Section;
});

jest.mock('./asos-logo.svg', () => {
  const AsosLogo = () => null;

  return AsosLogo;
});

jest.mock('./itv-logo.svg', () => {
  const ItvLogo = () => null;

  return ItvLogo;
});

describe('PlacesWorked', () => {
  const render = () => shallow(<PlacesWorked />);

  it('should render correctly', () => {
    expect(render()).toMatchSnapshot();
  });
});
