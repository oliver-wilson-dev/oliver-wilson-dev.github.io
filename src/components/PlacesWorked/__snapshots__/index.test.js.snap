// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlacesWorked should render correctly 1`] = `
<section
  className="sectionWrapper"
>
  <Section>
    <h3
      className="statement"
    >
      Proudly contributing towards projects in the fashion and television industries for over three years.
    </h3>
    <div
      className="companyLogoWrapper"
    >
      <AsosLogo
        className="companyLogo"
      />
      <ItvLogo
        className="itvLogo"
      />
    </div>
  </Section>
</section>
`;
